filebeat/fileset/fileset.go

```diff
@@ -153,10 +151,10 @@ func (fs *Fileset) readManifest() (*manifest, error) {
 }
 
 // evaluateVars resolves the fileset variables.
-func (fs *Fileset) evaluateVars(beatVersion string) (map[string]interface{}, error) {
+func (fs *Fileset) evaluateVars(info beat.Info) (map[string]interface{}, error) {
 	var err error
 	vars := map[string]interface{}{}
-	vars["builtin"], err = fs.getBuiltinVars(beatVersion)
+	vars["builtin"], err = fs.getBuiltinVars(info.IndexPrefix, info.Version)
```

Comment:
Perhaps we should pass `info` here as well, so `getBuiltinVars` has access to all the metadata, if it needs it in the future?

Reply:
done